go/vt/wrangler/materializer.go

```diff
@@ -1309,9 +1309,19 @@ func (mz *materializer) generateInserts(ctx context.Context, targetShard *topo.S
 		if mz.isPartial {
 			workflowSubType = binlogdatapb.VReplicationWorkflowSubType_Partial
 		}
+		var workflowType binlogdatapb.VReplicationWorkflowType
+		switch mz.ms.MaterializationIntent {
+		case vtctldatapb.MaterializationIntent_CUSTOM:
+			workflowType = binlogdatapb.VReplicationWorkflowType_Materialize
+		case vtctldatapb.MaterializationIntent_MOVETABLES:
+			workflowType = binlogdatapb.VReplicationWorkflowType_MoveTables
+		case vtctldatapb.MaterializationIntent_CREATELOOKUPINDEX:
+			workflowType = binlogdatapb.VReplicationWorkflowType_CreateLookupIndex
+		}
+
 		ig.AddRow(mz.ms.Workflow, bls, "", mz.ms.Cell, mz.ms.TabletTypes,
-			int64(mz.ms.MaterializationIntent),
```

Comment:
Replaced this with the explicit switch, since depending on enum order is really an implementation detail we should not (ab)use.